source/release-notes/v1.8-release-notes.rst

```diff
@@ -102,38 +103,149 @@ The Open OnDemand Portal :ref:`ood-portal-generator-configuration` file now supp
 Beta CCQ Scheduler support
 ..........................
 
+We now have beta `Cloudy Cluster`_ support for their ccq scheduler.  You can find
+:ref:`documentation on how to configure the cluster here <resource-manager-cloudy-cluster>`.
+
 Retain completed jobs panel for debugging
 .........................................
 
+When an interactive job completes, its panel will now remain in the view. This is to
+allow users and administrators to debug failed jobs by keeping the session id links
+in view.
+
 Batch Connect apps now support multiple clusters
 ................................................
 
+Batch connect applications now support multiple clusters in their configuration
+instead of just one.
+
+Where previous configuration look like this, with only a single cluster.
+
+.. code-block:: yaml
+
+   cluster: "my_cluster"
+
+Configurations will now accept an array like the example below.  This will create a
+dropdown select menu item at the top of the form where users can choose between
+the two options.
+
+.. code-block:: yaml
+
+   cluster:
+     - "my_cluster"
+     - "my_other_cluster"
+
+
+There are also other ways to configure which cluster to submit to.
+:ref:`Complete documentation can be found here. <configuring-cluster>`
+
 Batch Connect apps now support disabling caching on fields
 ..........................................................
 
+Batch connect apps can now enable or disable caching site wide, on a per app basis
+or on a per item basis.
+
+:ref:`See the full documentation on this feature here. <caching-from-items>`
+
 Extended view support for GridEngine clusters
 .............................................
 
-Torque and SLURM schedulers now supports QOS fields
+In the Active Jobs app, each job will have a dropdown where the user can see various details about their job such as walltime, Total CPUs, etc. 
+This functionality was recently implemented for SGE in v1.8 and corresponds to ood_core v0.12.0
+
+The data seen in extended view is extracted from the command ``qstat -j -r -xml``.
+
+Torque and Slurm schedulers now support QoS fields
 ...................................................
 
+Torque and Slurm schedulers now support the quality of service (QoS) fields.
+
+``qos`` is now an acceptable script attribute like so:
+
+.. code-block:: yaml
+
+   script:
+     qos: 'best'
+
+In Torque this translates to ``-l qos=best`` and in Slurm it is ``--qos best``.
+
 Shell hosts are now protected behind allowlists
 ...............................................
 
+Shell hosts are now protected behind allowlists. This means administrators now need
+to explicitly add hosts to an allowlist for users to be able to access them through
+the shell app.
+
+Any currently configured login host that is not hidden will be automatically allowed.
```

Comment:
Any login host of a cluster the cluster config will be allowed, assuming that cluster is not configured as "hidden" in the metadata section of the cluster config.